catalog/pipeline/fbc-release/0.11/fbc-release.yaml

```diff
@@ -0,0 +1,222 @@
+    - name: fbcPublishingCredentials
+      type: string
+      default: "fbc-publishing-credentials"
+      description: Secret used to publish the built index image
+    - name: requestUpdateTimeout
+      type: string
+      description: Max seconds to wait until the status is updated
+    - name: buildTimeoutSeconds
+      type: string
+      description: Max seconds to wait until the build finishes
+  workspaces:
+    - name: release-workspace
+  results:
+    - name: requestMessage
+      value: $(tasks.add-fbc-contribution-to-index-image.results.requestMessage)
+    - name: requestReason
+      value: $(tasks.add-fbc-contribution-to-index-image.results.requestReason)
+  tasks:
+    - name: verify-enterprise-contract
+      taskRef:
+        resolver: "bundles"
+        params:
+          - name: bundle
+            value: quay.io/hacbs-contract/ec-task-bundle:snapshot
+          - name: kind
+            value: task
+          - name: name
+            value: verify-enterprise-contract
+      params:
+        - name: IMAGES
+          value: $(params.snapshot)
+        - name: SSL_CERT_DIR
+          value: /var/run/secrets/kubernetes.io/serviceaccount
+        - name: POLICY_CONFIGURATION
+          value: $(params.enterpriseContractPolicy)
+        - name: STRICT
+          value: "1"
+    - name: add-fbc-contribution-to-index-image
+      workspaces:
+        - name: input
+          workspace: release-workspace
+      taskRef:
+        resolver: "bundles"
+        params:
+          - name: bundle
+            value: quay.io/hacbs-release/task-create-internal-request:0.5
+          - name: kind
+            value: task
+          - name: name
+            value: create-internal-request
+      params:
+        - name: pipelineRunName
+          value: $(context.pipelineRun.name)
+        - name: request
+          value: "iib"
+        - name: params
+          value:
+            - name: binaryImage
+              value: "$(params.binaryImage)"
+            - name: fromIndex
+              value: "$(params.fromIndex)"
+            - name: overwriteFromIndex
+              value: "$(params.overwriteFromIndex)"
+            - name: buildTags
+              value: "$(params.buildTags)"
+            - name: addArches
+              value: "$(params.addArches)"
+            - name: buildTimeoutSeconds
+              value: "$(params.buildTimeoutSeconds)"
+            - name: fbcFragment
+              value: '$(params.snapshot)'
+              jsonKey: ".components[0].containerImage"
+      runAfter:
+        - verify-enterprise-contract
+    - name: extract-requester-from-release
+      taskRef:
+        resolver: "bundles"
+        params:
+          - name: bundle
+            value: quay.io/hacbs-release/task-kubernetes-actions:0.2
+          - name: kind
+            value: task
+          - name: name
+            value: kubernetes-actions
+      params:
+        - name: script
+          value: |
+            set -x
```

Comment:
I would remove this. The `tee` in the bottom will give us some output already.

Reply:
It doesn't show the output because it is being set to a variable, that's why I added the `set -x`

Reply:
I mean, the output of the `tee` would be enough IMHO (maybe with some message) instead of a trace. I don't remember we letting the bash trace enabled in any script. But anyway, this is opinion. Keep the way you prefer.

Reply:
What I am trying to say is
`echo "foo" | tee /tmp/foo` prints `foo` as expected. `FOO=$(echo "foo" | tee /tmp/foo)` does not, without the `set -x`. So maybe the variable declaration needs improvement, but with just removing the `set -x`, you won't see the value in the log

Reply:
You are right. I did not notice the command was to assign to the var. Sometimes my  ADHD makes me waste my time due to the lack of observation.